releases/sui/2025-01-ITS-v1.1.0.md

```diff
@@ -80,55 +79,43 @@ kubectl exec genesis-0 -- axelard tx wasm execute "$ITS_HUB_ADDR" \
 --gas-adjustment 1.2
 ```
 
-#### Stagenet/Testnet:
+#### Stagenet/Testnet/Mainnet:
+
+ITS hub contract configuration in json file must include the following attributes per chain:
 ```bash
-node cosmwasm/submit-proposal.js its-hub-register-chains sui -t "Register ITS for Sui chain" -d "Register ITS for Sui chain at ITS Hub contract" --deposit <100000000 for stagenet and 2000000000 for testnet> \
--m '{
-  "register_chains": {
-    "chains": [
-      {
-        "chain": "sui",
-        "its_edge_contract": "'"$ITS_SUI_CHANNEL"'",
-        "truncation": {
-          "max_decimals_when_truncating": '"8"',
-          "max_uint": "'"18446744073709551615"'"
-        }
+"axelar": {
+  "contracts": {
+    ...
+    "InterchainTokenService": {
+      ...
+      "some-sui-chain": {
+        "maxUintBits": 64,
+        "maxDecimalsWhenTruncating": 6
       }
-    ]
+    }
+    ...
   }
-}'
+}
 ```
 
-#### Mainnet:
 ```bash
-node cosmwasm/submit-proposal.js its-hub-register-chains sui -t "Register ITS for Sui chain" -d <Description along with community proposal for ITS integration> --deposit 2000000000 \
--m '{
-  "register_chains": {
-    "chains": [
-      {
-        "chain": "sui",
-        "its_edge_contract": "'"$ITS_SUI_CHANNEL"'",
-        "truncation": {
-          "max_decimals_when_truncating": '"8"',
-          "max_uint": "'"18446744073709551615"'"
-        }
-      }
-    ]
-  }
-}'
+node cosmwasm/submit-proposal.js its-hub-register-chains sui -t "Proposal title" -d "Proposal description" --deposit <100000000 for stagenet and 2000000000 for testnet/mainnet>
```

Comment:
```suggestion
node cosmwasm/submit-proposal.js its-hub-register-chains sui -t "Proposal title" -d "Proposal description" --deposit $DEPOSIT_AMOUNT
```
Mention the env var setting above, instead of adding per network value within the cmd. We should just create simple tables for params by network